src/mame/drivers/pacman.cpp

```diff
@@ -1575,10 +1575,10 @@ static INPUT_PORTS_START( pacman )
 	PORT_DIPSETTING(    0x10, "15000" )
 	PORT_DIPSETTING(    0x20, "20000" )
 	PORT_DIPSETTING(    0x30, DEF_STR( None ) )
-	PORT_DIPNAME( 0x40, 0x40, DEF_STR( Difficulty ) )   PORT_DIPLOCATION("SW:7") // physical location for difficulty on puckman set is split-pad between R32 and C29
+	PORT_DIPNAME( 0x40, 0x40, DEF_STR( Difficulty ) )   // physical location for difficulty on puckman set is split-pad between R32 and C29
 	PORT_DIPSETTING(    0x40, DEF_STR( Normal ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( Hard ) )
-	PORT_DIPNAME( 0x80, 0x80, "Ghost Names" )           PORT_DIPLOCATION("SW:8") // physical location for ghostnames on puckman set is split-pad between C10 and C29
+	PORT_DIPNAME( 0x80, 0x80, "Ghost Names" )           // physical location for ghostnames on puckman set is split-pad between C10 and C29
 	PORT_DIPSETTING(    0x80, DEF_STR( Normal ) )
 	PORT_DIPSETTING(    0x00, DEF_STR( Alternate ) )
```

Comment:
Please make these either machine configuration or toggle inputs if they aren’t actually DIP switches.  MAME has a menu for toggle inputs now, so they aren’t completely opaque any more.

Reply:
Changed to machine configs